lib/emque/producing/publisher/rabbitmq.rb

```diff
@@ -5,6 +5,9 @@ module Emque
   module Producing
     module Publisher
       class RabbitMq < Emque::Producing::Publisher::Base
+        Emque::Producing::Message::PUBLISH_IGNORED_EXCEPTIONS << Bunny::Exception
+        Emque::Producing::Message::PUBLISH_IGNORED_EXCEPTIONS << Timeout::Error
```

Comment:
Should it be the other way around? The message reaching into the publisher?
Emque::Producing::Publisher::IGNORED_EXCEPTIONS?